scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -40,7 +40,16 @@ Positional arguments:
   dest-dir    Target directory; defaults to '$DEFAULT_DEST_DIR'.
   bundle-id   Support bundle ID; defaults to the local host name
               (`hostname`).
-
+  -c <config_url>  cluster config URL
```

Comment:
following options are placed under Positional arguments,
as these arguments are not positional.. can we move under optional arguments.